advocacy_docs/partner_docs/ImpervaDataSecurityFabric/07-Support.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Support and Logging Details'
+description: 'Brief details of the Support process and logging details'
```

Comment:
Change to "Brief details of the Support process and logging details for Imperva Data Security Fabric"